src/plugins/popup/popup.manager.js

```diff
@@ -73,11 +77,27 @@ export default class Popup {
 		this.element.removeAttr('tabindex');
 	}
 
+	isFocused() {
+		return this.state.active;
+	}
+	
+	keyDown(f) {
+		return new EventListener(this.element[0], new EventManager(this)).on('keydown', f);
+	}
+
 	resize(settings) {
 		this.element.css({
 			width: Math.min(settings.width, this.body.clientWidth - this.element.offsetLeft) + 'px',
 			height: Math.min(settings.height, this.body.clientHeight - this.element.offsetTop) + 'px'
 		});
 		this.event.emit('resize');
 	}
+
+	commands(commands) {
```

Comment:
what if the method'll be called sceral times? shortcutOff will be invalid, need to refactor this